install_config/aggregate_logging.adoc

```diff
@@ -300,6 +300,44 @@ when `openshift_logging_use_ops` is set to `true`.
 |Equivalent to `openshift_logging_kibana_replica_count` for Ops cluster
 when `openshift_logging_use_ops` is set to `true`.
 
+|`openshift_logging_es_allow_external`
+|Set to `true` to expose Elasticsearch as a route. Set to `false` by defaut.
+
+|`openshift_logging_es_hostname`
+|The external facing hostname to use for the route and the TLS server certificate.
+The default is set to `es`.
+
+|`openshift_logging_es_cert`
+|The location of the certificate Elasticsearch uses for the external TLS server
+cert. The default is a generated cert.
+
+|`openshift_logging_es_key`
+|The location of the key Elasticsearch uses for the external TLS server cert.
+The default is a generated key.
+
+|`openshift_logging_es_ca_ext`
+|The location of the CA cert for the cert Elasticsearch uses for the external TLS
+server cert. The default is the internal CA.
+
+|`openshift_logging_es_ops_allow_external`
+|Set to `true` to expose Elasticsearch as a route. Set to `false` by defaut.
+
+|`openshift_logging_es_ops_hostname`
+|The external facing hostname to use for the route and the TLS server certificate.
+The default is set to `es-ops`.
```

Comment:
`es-ops.` + openshift_master_default_subdomain.  For example, if `openshift_master_default_subdomain=example.test`, then the default value of `openshift_logging_es_ops_hostname` will be `es-ops.example.test`.